### ISSUE TITLE
S4058 is raised incorrectly in case of Xunit Assert.Contains

### ISSUE THREAD

**Issue by nhathongly** [SonarSource/sonar-dotnet#2099]
### Description

I got a [S4058](https://rules.sonarsource.com/csharp/RSPEC-4058) warning when using this Xunit method

`Assert.Contains(anyString, arrayOfString, StringComparer.OrdinalIgnoreCase)`

### Repro steps

```csharp
var returnedNames = new[] { "Any name", "Any other name" };
var expectedName = "Any name";

Assert.Contains(expectedName , returnedNames , StringComparer.OrdinalIgnoreCase);
```

### Expected behavior

No warning should be raised

### Actual behavior

An issue is raised as:

Warning	S4058	Change this call to 'Assert.Contains' to an overload that accepts a 'StringComparison' as a parameter.

Actually, XUnit does not have any overload of 'Assert.Contains' that takes 'StringComparision' as parameter.

### Related information

* SonarAnalyzer.CSharp - 7.8.0.7320
* Visual Studio - 15.8.9
* XUnit - 2.4.1

**Comment by andrei-epure-sonarsource** [comment]
This is the same problem that got reported for [S4056 ](https://rules.sonarsource.com/csharp/RSPEC-4056)in #2139 (so the fix should be similar, maybe we should be able to reuse the code)
